### ISSUE TITLE
No visible banner outline on high contrast mode 

### ISSUE THREAD

**Issue by aeroplanejane** [bbc/simorgh#8896]
**Describe the bug**

Banner outline not visible in high contract mode on AMP consent (clearly visible on privacy banner) 

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://5d28eb3fe163f6002046d6fa-lmnlzwmenv.chromatic.com/iframe.html?id=containers-consent-banner--amp&viewMode=story
2. View AMP banner 
3. Change colour preferences on Firefox 
4. Note that there is no outline on the banner 

**Expected behaviour**
Scenario: Changing colours

- Given I am using Windows High Contrast mode 
- And also when I change my colour preferences in FireFox
- When I view the consent banner
- Then a outline is visible for all buttons
- And I can see where the top of the compoment is e.g. there is a line/border 
- And I can see an affordance telling me there is content beneath the sticky buttons 

**Screenshots**
![Screenshot 2021-02-18 at 12 22 13](https://user-images.githubusercontent.com/22519682/108360312-f4e60a00-71e8-11eb-85a2-f0a62dd0eb1a.png)

**Desktop (please complete the following information):**

- Browser -  Firefox 
- PC high contrast 

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**
